relay/nakama/main.go

```diff
@@ -427,13 +427,13 @@ func initCardinalEndpoints(logger runtime.Logger, initializer runtime.Initialize
 func logCode(logger runtime.Logger, code int, format string, v ...interface{}) (string, error) {
 	err := eris.Errorf(format, v...)
 	logger.Error(eris.ToString(err, true))
-	return "", runtime.NewError(eris.ToString(err, true), code)
+	return "", runtime.NewError(err.Error(), code)
```

Comment:
changing this here because i dont think we wanna give clients the stack trace. just a simple error message.

Reply:
hmm this will be extremely convenient for development. We should have dev mode and prod mode for this.

Reply:
https://linear.app/arguslabs/issue/WORLD-573/make-a-nakama-dev-mode-that-returns-the-stack-traced-error